packages/component-submission/server/resolvers/submitManuscript.js

```diff
@@ -31,7 +31,10 @@ async function submitManuscript(_, { data }, { user, ip }) {
 
   manuscript.applyInput(manuscriptInput)
 
-  const sourceFile = manuscript.getSource()
+  logger.info(`Submitting ${manuscript.id}`)
+  logger.info(await manuscript.files)
```

Comment:
could use a full string describing what this is, in case it's `null` or something unrecognizable in a random log line